xbmc/addons/Scraper.cpp

```diff
@@ -816,6 +816,20 @@ void DetailsFromFileItem<CArtist>(const CFileItem &item, CArtist &artist)
     artist.discography.emplace_back(discoAlbum);
   }
 
+  int nVideolinks = item.GetProperty("artist.videolinks").asInteger32();
+  artist.videolinks.reserve(nVideolinks);
```

Comment:
I would include this line and the following for loop in `if (nVideoLinks > 0)`- saves some cpu cycles in case of no links available.

Reply:
Done.